content/en/blog/_posts/2022-08-15-local-storage-capacity-isolation-ga.md

```diff
@@ -0,0 +1,94 @@
+---
+layout: blog
+title: "Kubernetes 1.25: Local Storage Capacity Isolation Reaches GA"
+date: 2022-08-15
+slug: local-storage-capacity-isolation-ga
+---
+
+**Author:** Jing Xu (Google)
+
+Local ephemeral storage capacity isolation was introduced as a alpha feature in Kubernetes 1.7 and it went beta in 1.9 and with Kubernetes 1.25 we are excited to announce general availability(GA) of this feature.
+
+Pods use ephemeral local storage for scratch space, caching, and for logs. The lifetime of local ephemeral storage does not extend beyond the life of the individual pod. It is exposed to pods using the container’s writable layer, logs directory, and EmptyDir volumes. Before this feature available, there have been issues related to the lack of local storage accounting and isolation, such as Pods do not know how much local storage is available and cannot request guaranteed local storage. Local storage is a best effort resource and pods can be evicted due to other pods filling the local storage.
+
+The local storage capacity isolation feature allows users to manage local ephemeral storage in the same way as managing CPU and memory. It provides support for capacity isolation of shared storage between pods, such that a pod can be hard limited in its consumption of shared resources by evicting Pods if its consumption of shared storage exceeds that limit. It also allows setting ephemeral storage request for resource reservation. The limits and requests for shared ephemeral-storage are similar to those for memory and CPU consumption.
```

Comment:
We should link the feature to the concepts docs, if possible or if we have a feature doc, even better.

```suggestion
The [local storage capacity isolation feature](/docs/concepts/configuration/manage-resources-containers/#local-ephemeral-storage) allows users to manage local ephemeral storage in the same way as managing CPU and memory. It provides support for capacity isolation of shared storage between pods, such that a pod can be hard limited in its consumption of shared resources by evicting Pods if its consumption of shared storage exceeds that limit. It also allows setting ephemeral storage requests for resource reservation. The limits and requests for shared `ephemeral-storage` are similar to those for memory and CPU consumption.
```